release-notes/opensearch-documentation-release-notes-2.5.0.md

```diff
@@ -0,0 +1,21 @@
+# OpenSearch Documentation Website 2.5.0 Release Notes
+
+The following page provides a list of updates made to OpenSearch documentation for OpenSearch 2.5.0.
+
+## New documentation for 2.5.0
+
+- Add Query String for rollups [#2428](https://github.com/opensearch-project/documentation-website/pull/2428)
+- Updates to Security Analytics documentation [#2408](https://github.com/opensearch-project/documentation-website/pull/2408)
+- Add new Rules documentation that covers YAML Editor view [#2407](https://github.com/opensearch-project/documentation-website/pull/2407)
+- Adds documentation for Admin UI index operations [#2403](https://github.com/opensearch-project/documentation-website/pull/2403)
```

Comment:
Should we rename this to match our changes? or are we sticking with the PR name?

Reply:
@ariamarble Do you have the ability to still change the PR name? If not, I think it's fine to leave it as is.